build/travis/job2_AppImage/bintray.sh

```diff
@@ -87,8 +87,14 @@ case "${ARCH}" in
   i686|i386|i[345678]86 )
     SYSTEM="${ARCH} (32 bit Intel/AMD)"
     ;;
-  *[Aa][Rr][Mm]* )
-    SYSTEM="${ARCH} (32 bit ARM)"
+  armel )
+    SYSTEM="${ARCH} (32-bit ARMv4t and above, software float)"
+    ;;
+  armhf )
+    SYSTEM="${ARCH} (32-bit ARMv7 and above, hardware float)"
+    ;;
+  aarch64 )
+    SYSTEM="${ARCH} (64-bit ARMv8 and above)"
```

Comment:
Hmm, now we're getting really technical! That might be a bit much for some users. Something like this might be better:

- armel (old 32-bit ARM)
- armhf (new 32-bit ARM)
- aarch64 (64-bit ARM)

Then we could have a special section in the handbook where we explain the difference. Something like "how to identify your ARM processor's architecture". Alternatively, we could go down the Ubuntu route and simply not support armel at all...

Feel free to change my strings to 32-bit Intel/AMD if you prefer dashes.